MyClassroom/Views/Shared/_Layout.cshtml

```diff
@@ -25,6 +25,15 @@
                         <li class="nav-item">
                             <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="Privacy">Privacy</a>
                         </li>
+                        @if (!User.Identity?.IsAuthenticated ?? true)
```

Comment:
Are we going to support log-out operation?

Reply:
Yes, I will have to implement logout as well